app/scripts/views/mixins/password-mixin.js

```diff
@@ -45,6 +46,29 @@ define(function (require, exports, module) {
         // IE8 blows up when changing the type of the input field. Other
         // browsers might too. Ignore the error.
       }
+    },
+
+    onPasswordKeypress: function (event) {
+      var val = this.getElementValue('.password').length;
+      if (val > 6) {
+        this.setPasswordWarning(true);
+      } else {
+        this.setPasswordWarning(false);
+      }
+    },
+
+    setPasswordWarning: function (isVisible) {
+      try {
+        var passwordhelp = this.$('.input-help');
+        if (isVisible) {
+          passwordhelp.css('opacity', '0');
+        } else {
+          passwordhelp.css('opacity', '1');
+        }
+      } catch(e) {
+        // IE8 blows up when changing the type of the input field. Other
```

Comment:
You could probably remove the try/catch from here because we are not changing input type.